cmd/getenvoy/main.go

```diff
@@ -17,13 +17,12 @@ package main
 import (
 	"os"
 
+	"github.com/tetratelabs/getenvoy/pkg/binary/envoy/globals"
 	"github.com/tetratelabs/getenvoy/pkg/cmd"
-	cmdutil "github.com/tetratelabs/getenvoy/pkg/util/cmd"
 )
 
 func main() {
-	root := cmd.NewRoot()
-	if err := cmdutil.Execute(root); err != nil {
+	if cmd.Execute(cmd.NewRoot(&globals.GlobalOpts{})) != nil {
```

Comment:
better print the error returned by cmd.Execute?

Reply:
it is probably not easy to tell but `cmd.Execute(cmd)` does this. We have some special case logging we do which is different `cmd.Execute()` one day I hope we can use a different cli lib

Reply:
I think maybe I can rename the import as the word clash is confusing